### DISCUSSION TITLE
V3 wont load anything from SD card

### DISCUSSION THREAD

**Discussion by AlexM19** [gtxaspec/wz_mini_hacks#446] [Q&A]
Hi,

I bought 4 v3 cam and manually updated three of them to version 4.36.9.139 from the SD card and then loaded the hack without any issue. The rtsp is working perfectly. 

But the fourth one would not update from the SD card at all, no matter how long I pressed the button. I then updated it trough the android app to version 4.36.9.139. But just like the update, the hack would not load. 

I know the card is working fine since I enable the recording on it from the android app. I tried multiple other card and even made the cam format the card itself from the app then placed the file on it. It like the bootloader on this unit is not working properly. 

Any of you guys have experienced this before?

Thanks

**Comment by endertable** [comment]
The card might be formatted as vfat vs FAT32, especially if you used the app to format it.  The card MUST be FAT23 for the hack to work.  One thing you can try to eliminate if issue is with cam vs. issue is with card is to swap the card of a working cam.  If the 4th cam then starts working then you have a bad or poorly formatted card.

**Reply by AlexM19** [reply]
Unfortunately I tried all of this to no avail. 

I did a bit more research and it seems some people had the same problem but with the Pan instead of the V3:
https://forums.wyze.com/t/cant-flash-firmware-to-cam-pan/95238/64
https://github.com/EliasKotlyar/Xiaomi-Dafang-Hacks/issues/1563

The guy posting on the Dafang project actually found a solution (for the Pan) here:
https://github.com/agent86ix/wyze-cam-pan-sd-flash-fix

Although, I am not sure if this would work with my v3. Mainly, I am not quite sure what bootloader to use.

**Reply by virmaior** [reply]
to use that solution,
1. you need to find a bootloader for v3 not v2/pan-v1
2. downgrade your camera's firmware quite a few versions so that the wyzeUpdater will actually be able to run. It's been a while but I think you'll need to do some DNS spoofing to trick the camera into accepting an update from what's not actually wyze's server.

Separately, I think there'd be some interest in getting a copy of the bootloader that doesn't work to figure out if you there's a different filename or something.

**Reply by AlexM19** [reply]
Thanks to have taken the time to awnser me. Your comments are inline with my original plan. But during the holidays I explain my problem with my brother in law, and he offers me to exchange the faulty camera with one of his since he did not intended to use the hack.

So I went the lazy way and just bypass the problem all together... 

Thanks again for the insight.

**Comment by endertable** [comment]
Be careful, I believe those posts are referencing the V1 Pan Cam.  That is more on the same kernel level as maybe the V2 cam but not the V3.  I would not use any uboot images from those.  A V2 Pan Cam may be more on the same playfield as the V3.

**Comment by akivacp** [comment]
Try putting everything under SD_ROOT in the root level of the SD card. 
For example, if the SD Card is your E: drive,
your files should look like this when you open E:
```
wz_mini (directory)
factory_t31_ZMC6tiIDQN
```
